packages/@aws-cdk/aws-servicediscovery/lib/http-namespace.ts

```diff
@@ -0,0 +1,47 @@
+import cdk = require('@aws-cdk/cdk');
+import { CfnHttpNamespace } from './servicediscovery.generated';
+
+export interface HttpNamespaceProps {
+  /**
+   * A name for the HttpNamespace.
+   *
+   * @default CloudFormation-generated name
+   */
+  name: string;
+
+  /**
+   * A description of the namespace.
+   */
+  description?: string;
+}
+
+/**
+ * Define a Service Discovery HTTP Namespace
+ */
+export class HttpNamespace extends cdk.Construct {
+  /**
+   * A domain name
+   */
+  public readonly name: string;
```

Comment:
This variable doesn't seem to be assigned. Unsure why TypeScript lets you get away with this.